.github/workflows/smoke-test-prod-podman.yml

```diff
@@ -0,0 +1,108 @@
+name: Production Smoke Test Podman (via compose.prod.yml)
+
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  smoke-test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Copy environment file
+        run: cp example.prod.env .prod.env
+
+      - name: Add SECRET_KEY to environment file
+        run: |
+          echo -e "\nSECRET_KEY=${{ secrets.SECRET_KEY }}" >> .prod.env
+      - name: Set up Podman
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install podman
+          pip install podman-compose
+
+          echo "$HOME/.local/bin" >> $GITHUB_PATH
+
+          podman --version
+          which podman-compose
+
+      - name: Start containers
+        run: podman-compose -f compose.prod.yml up -d
+
+      - name: Wait for database to be ready
+        run: |
+            echo "Waiting for database to be ready..."
+            timeout=90
+            elapsed=0
+
+            # Wait for database to start accepting connections
+            until [ $elapsed -ge $timeout ] || podman exec $(podman ps -q --filter name=db) mysql --no-defaults -u records-user -precords-password -e "SELECT 1" &>/dev/null; do
+            sleep 10
+            elapsed=$((elapsed+10))
+            echo "Database still initializing... ($elapsed/$timeout seconds)"
+            done
+
+            if [ $elapsed -ge $timeout ]; then
+            echo "Database initialization timed out"
+            podman-compose -f compose.prod.yml logs db
+            exit 1
+            fi
+
+            echo "Database is ready"
+
+      - name: Wait for app to start
+        run: |
```

Comment:
Same comment as above. Use two spaces of indentation because that's how it is in all the other `run:` blocks. The `until` block and the first `if` block are not indented , either.